pages/index.html

```diff
@@ -19,6 +19,169 @@
 
     <!-- category section start -->
 
+    <section class="py-12 py-lg-15">
+      <div class="container mb-8">
```

Comment:
@kira134679  午安，有看到一個小地方需要修改，還有兩個問題想請問Tomo。
 標題的margin-bottom需要再加上電腦版的設定:
```suggestion
      <div class="container mb-8 mb-lg-12">
```